### PR TITLE
Switch to env based configuration + remove `"init"` request

### PR DESCRIPTION
**Pull request by wkozyra95**
Switch to env base config

### Why envs

Why not init request:
- some values need to be set before e.g. API PORT, so it requires using both methods of configuration
- it's very non-standard config method, even if it was the best option it could be perceived as hack.

Why not config files:
- hard to work with in prod environment, to add config inside k8s or any other container-based platform you would need to: create some volume entity from that file, attach/mount that volume to the container, and pass some env that defines the path where compositor should look for the file. 
- hard to work with in a dev environment. To tests some other configuration you need to modify a file. If we want to persist the config we can always use `.env` file.

### CODE REVIEW

File: docs/pages/deployment/configuration.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+See `-loglevel` option in [FFmpeg documentation](https://ffmpeg.org/ffmpeg.html).
+
+### `LIVE_COMPOSITOR_WEB_RENDERER_ENABLE`
+
+Enable web rendering capabilities. With this option disabled, you can not use [`WebView` components](../api/components/WebView) or register [`WebRenderer` instances](../api/renderers/web).
+
+### `LIVE_COMPOSITOR_WEB_RENDERER_GPU_ENABLE`
+
+Disable GPU support inside the embedded Chromium instance.
```
Comment (noituri) [inline docs/pages/deployment/configuration.md:51]:
```suggestion
Enable GPU support inside the embedded Chromium instance.
```

The env's name contains `ENABLE` so I think it makes sense to also reflect that in the description

**Review by WojciechBarczynski — APPROVED** [review]
LGTM

File: docs/pages/deployment/configuration.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+### `LIVE_COMPOSITOR_LOGGER_FORMAT`
+
+Logger format. Supported options:
+- `json`
+- `compact`
+- `pretty`
+
+:::warning
+This option does not apply to logs produced by `FFmpeg` or the embedded Chromium instance used for web rendering.
+:::
```
Comment (jerzywilczek) [inline docs/pages/deployment/configuration.md:32]: what does `:::` do?

**Review by jerzywilczek — APPROVED** [review]
:rocket:

File: docs/pages/deployment/configuration.md (hunk 2 above)
Comment (noituri) [reply docs/pages/deployment/configuration.md:32]: image

^ This

**Review by noituri — APPROVED** [review]

File: docs/pages/deployment/configuration.md
Comment (WojciechBarczynski) [inline docs/pages/deployment/configuration.md:1]: I think all those env vars should have `required` / `optional` next to them (with required being the first ones). For `optional` values provide defaults.

Comment (WojciechBarczynski) [reply docs/pages/deployment/configuration.md:1]: You can also do two separate sections for that

Comment (WojciechBarczynski) [reply docs/pages/deployment/configuration.md:1]: I'm pretty sure that output framerate is (or at least should be) required

**Pull request merged**